packages/playground/src/components/profile_manager/ConnectWallet.vue

```diff
@@ -124,16 +125,18 @@
           label="Email"
           placeholder="email@example.com"
           :loading="loadEmail"
-          :disabled="creatingAccount || connecting || loadEmail || mnemonicInput?.status !== ValidatorStatus.Valid"
+          :disabled="isEmailDisabled"
           autocomplete="off"
+          @blur="() => emailInput?.validate()"
+          @update:model-value="() => emailInput?.validate()"
```

Comment:
I think this is the actual behavior of the input validator. Why do we need to specify that?

Reply:
After handling the disabled part, it doesn't work, so I handled it this way